doc/migration/version-upgrade-guide.md

```diff
@@ -940,9 +983,22 @@ func serveCRMetrics(cfg *rest.Config, operatorNs string) error {
 
 ### Breaking changes
 
-#### Remove Ansible container sidecar
+#### `TestCtx` in `pkg/test` has been deprecated
+ 
+ The type name `TestCtx` in `pkg/test` has been deprecated and renamed to `Context`. Users of the e2e framework should do the following:
+ 
+ - Replace `TestCtx` with `Context`
+ - Replace `NewTestCtx` with `NewContext`
+
+#### Scorecard only supports YAML config files
+  
+The scorecard feature now only supports YAML config files. Any config file with other extension is deprecated and should be changed for the YAML format. For further information see [`scorecard config file`](https://github.com/operator-framework/operator-sdk/blob/v0.16.x/doc/test-framework/scorecard.md#config-file)
```

Comment:
Are other extensions deprecated or no longer supported?

```suggestion
The scorecard feature now only supports YAML config files. Config files with other extensions are no longer supported and should be changed to the YAML format. For further information see [`scorecard config file`](https://github.com/operator-framework/operator-sdk/blob/v0.16.x/doc/test-framework/scorecard.md#config-file)
```

Reply:
HI @joelanford, 

I am not sure if I got your question: 
> Are other extensions deprecated or no longer supported?

See that:

> The scorecard feature now only supports YAML config files. Config files with other extensions are no longer supported ...